src/transformers/onnx/convert.py

```diff
@@ -62,90 +62,172 @@ def check_onnxruntime_requirements(minimum_version: Version):
         )
 
 
-def export(
-    tokenizer: PreTrainedTokenizer, model: PreTrainedModel, config: OnnxConfig, opset: int, output: Path
+def export_pytorch(
+    tokenizer: PreTrainedTokenizer,
+    model: PreTrainedModel,
+    config: OnnxConfig,
+    opset: int,
+    output: Path,
 ) -> Tuple[List[str], List[str]]:
     """
-    Export a PyTorch backed pipeline to ONNX Intermediate Representation (IR
+    Export a PyTorch model to an ONNX Intermediate Representation (IR)
 
     Args:
-        tokenizer:
-        model:
-        config:
-        opset:
-        output:
+        tokenizer: The pipeline's tokenizer.
+        model: The Pytorch (PreTrainedModel) model that will be converted.
+        config: The ONNX config object associated to the model that will be converted.
+        opset: The actual version of the ONNX operator set to use
+        output: Path where will be stored the generated ONNX model
```

Comment:
Here's a suggestion to make the docstring style be closer to the rest of the codebase:

```suggestion
        tokenizer ([`PreTrainedTokenizer`]):
            The tokenizer used for encoding the data.
        model ([`PreTrainedModel`]):
            The model to export.
        config ([`~onnx.config.OnnxConfig`]):
            The ONNX configuration associated with the exported model.
        opset (`int`):
            The version of the ONNX operator set to use.
        output (`Path`):
            Directory to store the exported ONNX model.
```